File: apps/problems/two-sum/boilerplate-full/function.js
## USER_CODE_HERE ##

const fs = require("fs");
const input = fs.readFileSync(0, "utf-8").trim().split(/\s+/);

const a = Number(input[0]);
const b = Number(input[1]);

const result = twoSum(a, b);
console.log(result);

File: apps/problems/two-sum/boilerplate-full/function.rs
use std::io::{self, Read};

## USER_CODE_HERE ##

fn main() {
    let mut input = String::new();
    io::stdin().read_to_string(&mut input).unwrap();
    let mut iter = input.split_whitespace();

    let a: i32 = iter.next().unwrap().parse().unwrap();
    let b: i32 = iter.next().unwrap().parse().unwrap();

    let result = twoSum(a, b);
    println!("{}", result);
}